inventory/hosts

```diff
@@ -108,11 +108,12 @@ uk-staging2
 [us-prod]
 openfoodnetwork.net ansible_host=52.3.171.208
 
-[us-staging]
+[us-migrate]
+migrate.openfoodnetwork.net ansible_host=64.227.102.29
```

Comment:
so we'll keep both domains during the migration and then do switch over?

Reply:
Yeah, I was basing it on this: https://github.com/openfoodfoundation/ofn-install/pull/611/files